### PR TITLE
changes for the Downloads feature 

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

File: projects/project-player-library/src/lib/constants/actionConstants.ts [hunk 1]
```diff
@@ -92,5 +92,17 @@ export const actions = {
...
         },
-    ]
+    ],
+    PROJECT_DOWNLOAD : {
+            title: "DOWNLOAD",
+            icon: "cloud_download",
+            action: "download",
+            color: "primary-icon"
+    },
+    PROJECT_DOWNLOADED : {
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/constants/actionConstants.ts]: This is already declared in line 34

File: projects/project-player-library/src/lib/constants/actionConstants.ts [hunk 2]
```diff
@@ -92,5 +92,17 @@ export const actions = {
             action:"delete",
             color: "primary-icon"
         },
-    ]
+    ],
+    PROJECT_DOWNLOAD : {
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/constants/actionConstants.ts]: Why are we creating it as a new key
Cant we add this value in the PROJECT_ACTIONS array itself?

File: projects/project-player-library/src/lib/constants/actionConstants.ts (hunk 1 above)
Comment (farhanp1502) [reply projects/project-player-library/src/lib/constants/actionConstants.ts]: reused the same download action

File: projects/project-player-library/src/lib/constants/actionConstants.ts (hunk 2 above)
Comment (farhanp1502) [reply projects/project-player-library/src/lib/constants/actionConstants.ts]: made the changes

File: projects/project-player-library/src/lib/constants/actionConstants.ts [hunk 3]
```diff
@@ -33,7 +39,7 @@ export const actions = {
     },
     DOWNLOADED_ACTION : {
         title: "DOWNLOADED",
-        icon: "check_circle",
+        icon: "cloud_download",
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/constants/actionConstants.ts]: change to icon to the previous one
"check_circle"

File: projects/project-player-library/src/lib/pages/details-page/details-page.component.ts [hunk 4]
```diff
@@ -201,6 +206,11 @@ export class DetailsPageComponent implements OnInit {
         options.push(actions.CERTIFICATE_ACTION)
       }
     }
+    else {
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/pages/details-page/details-page.component.ts]: Why are we doing this in the else block?
Check how SYNC icon is handled

Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/details-page/details-page.component.ts]: done

File: projects/project-player-library/src/lib/constants/actionConstants.ts (hunk 3 above)
Comment (farhanp1502) [reply projects/project-player-library/src/lib/constants/actionConstants.ts]: okay

**Review by Vinod-V3 — APPROVED** [review]

**Pull request merged**